app/src/main/java/com/concordium/wallet/ui/cis2/TokensAddAdapter.kt

```diff
@@ -66,7 +66,7 @@ class TokensAddAdapter(
         val tokenBalance = CurrencyUtil.formatGTU(
             token.totalBalance,
             false,
-            token.tokenMetadata?.decimals ?: 6
+            token.tokenMetadata?.decimals ?: 0
```

Comment:
Incidental fix I stumbled into (tokens without decimals should be treated as integers)